src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_validators.py

```diff
@@ -117,7 +117,8 @@ def test_resource_types_type(self):
     def test_services_type(self):
         input = "ttfqbqtf"
         actual = str(services_type(self.loader)(input))
-        if supported_api_version(self.cli, ResourceType.DATA_STORAGE, max_api='2016-05-31'):
+        if supported_api_version(self.cli, ResourceType.DATA_STORAGE, max_api='2016-05-31') or \
+           supported_api_version(self.cli, ResourceType.DATA_STORAGE, min_api='2017-07-29'):
```

Comment:
Don't need. But don't let this blocking this PR. Make another change to fix it later.